packages/lesspass-pure/src/i18n/en.json

```diff
@@ -53,6 +53,7 @@
   "SorryCopy": "Sorry, copying only works in modern browsers.",
   "UpdateYourSearch": "Please try broadening your search.",
   "Username": "Username",
+  "Use plain password": "Use plain password (not recommended)",
```

Comment:
I'm not sure why I don't get the **full** translation string of the _Use plain password_ checkbox in English. If I switch my browser in French I get it but not in English. Any hints?

Reply:
Hum, Maybe it use `$t("Use plain password")` directly. Ignore the translation in this PR, I will look at it at the end.
I will run automatic scripts to do it for us.

Reply:
Sounds good!